src/header-navigation/styled-components.js

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index';
+import {ALIGN} from './index';
+
+export const Root = styled('nav', props => {
+  const {$theme} = props;
+  const {
+    sizing: {scale500},
+    typography: {font400},
+    colors: {mono500},
```

Comment:
instead of depending on exact colors, can you use theme variables like `colors.foreground` to make sure the navigation component will work with the dark theme too?

Reply:
ok